h/tasks/indexer.py

```diff
@@ -48,34 +45,13 @@ def sync_annotations(limit):
     log.info(dict(counts))
     newrelic.agent.record_custom_metrics(
         [
-            (f"Custom/SyncAnnotations/Queue/{key}", value)
+            (f"Custom/JobQueue/Name/sync_annotation/Result/{key}", value)
```

Comment:
Matches the `/Name/{name}/` format used by the new `JobQueueMetrics.metrics()` method when it automatically generates per-name counts